README.md

```diff
@@ -37,9 +62,13 @@ Provide a custom User-Agent:
 KnowSheet> HTTP(GET("http://httpbin.org/get").UserAgent("KnowSheet-REPL/1.0.0")).body
 ```
 
-To quit REPL, hit `Ctrl+C` twice:
+Provide extra HTTP headers:
+```
+KnowSheet> HTTP(GET("http://httpbin.org/get?query=1", HTTPHeaders().Set("Custom", "Header"))).body
+KnowSheet> HTTP(POST("http://httpbin.org/post", "BODY", "text/plain", HTTPHeaders().Set("Custom", "Header"))).body
```

Comment:
I tried to make this one multiline, but when copy-pasted, the REPL executed it line-by-line (with errors on incomplete lines and partial success on complete lines.